AndroidSDK/src/main/java/com/leanplum/LeanplumPushService.java

```diff
@@ -35,6 +35,7 @@
 import android.os.Bundle;
 import android.support.v4.app.NotificationCompat;
 import android.text.TextUtils;
+import android.widget.RemoteViews;
```

Comment:
I think this one not from this code change.

Reply:
yeah it looks like. no idea how it got through.